src/outfit.c

```diff
@@ -2957,3 +3041,83 @@ void outfit_free (void)
    array_free(outfit_stack);
    array_free(license_stack);
 }
+
+t_os_stat os_opts(const char *name, const char *unit, int color, double threshold, int hidezero, int precision)
+{
+   t_os_stat opts = {0};
+   opts.name = name;
+   opts.unit = unit;
+   opts.color = color;
+   opts.color_threshold = threshold;
+   opts.precision = precision;
+   opts.hide_zero = hidezero;
+   return opts;
+}
+
+#define ItoL(i) OUTFIT_SHORTDESC_MAX-i
+
+/**
+ * @brief Print an outfit double statistic line
+ *
+ */
+int os_printD( char *buffer, int i, double value, t_os_stat opts)
+{
+   if (opts.hide_zero && value == 0)
```

Comment:
probably better to do `fabs(value) < 1e-5` or something like that instead of `value == 0`